Makefile

```
@@ -1,26 +1,32 @@
+# Use uv for all operations
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)